README.md

```diff
@@ -17,3 +17,24 @@ Download the latest [release](https://github.com/cdr/coder-cli/releases):
 cd ~/go/bin
 tar -xvf ~/Downloads/coder-cli-linux-amd64.tar.gz
 ```
+
+## Usage
+
+```
+$ coder --help
+Usage: coder [subcommand] [flags]
```

Comment:
We shouldn't really maintain the output of commands in the README since it changes frequently. We should just list there is a --help flag and remove the output.

Reply:
I find it very helpful from a user perspective in understanding the basic functionality within the first 10 seconds of interacting with the repo. I'll add a CI check as part of #63 .

Reply:
On second thought, let's table this for now and revisit when the state of the documentation can be fully considered.